jinahub/indexers/query/vector/FaissIndexer/__init__.py

```diff
@@ -72,8 +77,54 @@ def __init__(self,
         self.distance = distance
         self.normalize = normalize
         self.nprobe = nprobe
+        self.on_gpu = on_gpu
+
+        self.default_top_k = default_top_k
+        self.traverse_path = traverse_path
+
+        self.logger = get_logger(self)
```

Comment:
We can maybe use directly `JinaLogger`?

Reply:
Wanted to provide an utility that automatically knows what the naming convention of the logger should be. All executors/indexers should have the same naming convention for their loggers. I see some that use class name, or metas.name, or other strings. It should be unified